integerations/express.ts

```diff
@@ -0,0 +1,117 @@
+import express from "express";
+import Keploy from "../src/keploy"; 
+import { Request, Response, NextFunction } from 'express';
+
+type dependency = {
+    name: string,
+    type: string,
+    meta: object,
+    data: object[]
+}
+
+type Kctx = {
+    mode: string | undefined,
+    testId: string | undefined,
+    deps: dependency[] | undefined
+}
+
+class Context {
+  static _bindings = new WeakMap<Request, Context>();
+
+  public kctx: Kctx;
+
+  constructor (mode?: string, testId?: string, deps?: dependency[]) {
+    this.kctx= {
+        mode,
+        testId,
+        deps
+    }
+  }
+  // bind sets an empty Context for req as a key in _bindings.
+  static bind (req: Request) : void {
```

Comment:
We use prettier formatting, install vscode-eslint plugin for it to work

Reply:
installed and applied.